code/BagsApi.cs

```diff
@@ -29,6 +30,39 @@ public BagsApi()
 			return JsonConvert.DeserializeObject<Product>(content);
 		}
 
+		public virtual async Task<List<Product>> GetProductsByTags(String[] tags)
```

Comment:
Versatility: Change `String[]` to `IEnumerable<String>`.  It is more versatile than `String[]` as you can pass in an array, a list, a set, etc.